pyproject.toml

```diff
@@ -62,6 +62,7 @@ markers = [
   "group_a: fast test subset a",
   "group_b: fast test subset b",
   "slow_tests: tests that take a long duration to complete",
+  "dragon: dragon tests",
```

Comment:
pedantic/optional request - could we replace defining something with itself by saying "tests that must be executed in a dragon runtime"